docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,126 @@
+# Tutorial
+
+The wordpress-k8s charm helps deploy a horizontally scalable WordPress application with ease and
+also helps operate the charm by liaising with the Canonical Observability Stack (COS). This
+tutorial will walk you through each step of deployment to get a basic WordPress deployment.
+
+### Prerequisites
+
+To deploy wordpress-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the tutorial model
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up a new model with the following command.
+
+```
+juju add-model wordpress-tutorial
+```
+
+### Deploy wordpress-k8s charm
+
+Deployment of WordPress requires a database integration. The preferred method in juju is through an
+integration. In this case, a `mysql` interface is required by the wordpress-k8s charm and
+hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.
+
+Start off by deploying the wordpress charm. By default it will deploy the latest stable release of
+the wordpress-k8s charm.
+
+```
+juju deploy wordpress-k8s
+```
+
+### Deploy and relate database
+
+#### Database relation
+
+The following commands deploys the mysql-k8s charm and relates wordpress-k8s charm through its mysql
+interface.
+
+```
+juju deploy mysql-k8s \
+    --config mysql-interface-user=wordpress-tutorial-user \
+    --config mysql-interface-database=wordpress-tutorial-database
+juju relate wordpress-k8s mysql-k8s
```

Comment:
The current doc has this command "juju relate wordpress-k8s mariadb:mysql".  The ":mysql" isn't it necessary?

Reply:
Indeed, fixed! :)